### PR TITLE
fix nav icons

### LINKED ISSUE #81: Update Nav Bar Alignment
The nav bar currently shows icons and font that is not correctly aligned/sized. 

- Adjust sizing of the icons to be uniform
- Vertically center icons
- Adjust font size so that all text fits on a single line
- Align text along bottom of container
- Rename `INVENTORY` to `CARRIERS`

### PR DESCRIPTION
**Pull request by Rgaliant**
Resolves #081 

### Description
Fixed the CSS for the icons in the navbar

### Type of change

* Bug fix (non-breaking change which fixes an issue)
https://github.com/rubyforgood/babywearing/issues/81
### How Has This Been Tested?

On homepage all icons are same size and aligned to the bottom
### Screenshots

Screen Shot 2019-10-10 at 9 28 23 AM

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -33,31 +33,73 @@ $navbar-brand-padding-y: 8px;
...
-.background-purple {background-color: $bw-color-purple}
-.background-dark-purple {background-color: $bw-color-dark-purple}
-.background-slate {background-color: $bw-color-slate}
-.background-blue {background-color: $bw-color-blue}
-.background-darkest-purple {background-color: $bw-color-darkest-purple}
-.navbar-text .pull-right {position: fixed}
+.navbar-brand img {
+  height: 84px;
+}
+// .nav-icon img {
```
Comment (dixpac) [inline app/assets/stylesheets/application.scss]: ✂️commented out code

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -33,31 +33,73 @@ $navbar-brand-padding-y: 8px;
...
+.navbar-text .pull-right {
+  position: fixed;
+}
+
+.nav-item {
+  display: flex;
+  align-items: center;
+}
 
+#nav-first img {
```
Comment (dixpac) [inline app/assets/stylesheets/application.scss]: Not sure that using `ids` inside the css is a good approach, personally I would keep `ids` outside the css.
https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (Rgaliant) [reply app/assets/stylesheets/application.scss]: 👍

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (Rgaliant) [reply app/assets/stylesheets/application.scss]: 👍changed to classes

**Comment by pollygee** [comment]
@dixpac can you take another look and merge this if it's good to go?  Thanks!!

**Review by dixpac — APPROVED** [review]
👍

**Pull request merged**